src/frontend/js/utils/StringHelper/index.ts

```diff
@@ -19,4 +19,15 @@ export class StringHelper {
       .join('')
       .toUpperCase();
   }
+
+  static toColor(string: string, saturation = 50, lightness = 55): string {
+    let hash = 0;
+    for (let i = 0; i < string.length; i++) {
+      // eslint-disable-next-line no-bitwise
+      hash = string.charCodeAt(i) + ((hash << 5) - hash);
+      // eslint-disable-next-line no-bitwise
+      hash &= hash;
+    }
+    return `hsl(${hash % 360}, ${saturation}%, ${lightness}%)`;
+  }
 }
```

Comment:
What do you think to reduce the hue available to homogeneize color palette ?
<img width="693" alt="image" src="https://github.com/openfun/richie/assets/9265241/b36c6b56-e067-4bed-aed6-805fbfed344a">

Reply:
I don't like to loose colors differences, it's the whole point of these default color, to be different.

Reply:
Too many discution about this small improvement proposal, i moved it in a [dedicated PR](https://github.com/openfun/richie/pull/2297)